js/client-chat.js

```diff
@@ -808,7 +808,25 @@
 					}
 				}), 'text');
 				return false;
-
+				
+			case 'regdate':
+				if (!target) target = app.user.get('userid');
+				target = toId(target);
+				var self = this;
+				$.get("http://pokemonshowdown.com/users/" + target + ".json",
```

Comment:
This will be a security error in HTTPS, but I don't know enough about XHR to know if this is considered cross-origin. It's also unnecessary database load, you're getting a ladder list for no reason. You probably want to do it more similarly to how `/rank` is currently implemented.